### ISSUE TITLE
Cool, yet easy way to show quotes

### ISSUE THREAD

**Issue by RogerFK** [TwidereProject/Twidere-Android#526]
So, you can skip that story. There's a TL;DR
I was wondering how could I see my quotes in another tab. I started by putting a search tab with `twitter.com/myusername` but then I realized that it showed my photos and videos. Also really cool, so you can get your own videos by doing `twitter.com/myusername from:myusername`. And then I thought wow, wouldn't it be amazimg to exclude your own tweets?

So I found out that if you create a search tab with 
`twitter.com/Username -from:@Username`
Then you will find quoted tweets upto 3 days. Tuck Fwitter. So yeah, adding it would be great for new users, or making a Twidere Tips and Tricks series. Yeah that's a lot of YouTube monayy boyzz. Oh, if you do that YouTube video mention me, please. And Mariotaku, thanks if you add it. Have a nice day!

TL;DR: by searching `twitter.com/username -from:@username` you can get quoted tweets at you. Yeah or even videos/photos from you other people upload.

**Comment by StefanBrand** [comment]
Thanks a lot for sharing this here! :D It even gives me quotes from 9 days ago (well there aren't too many).
